gravitee-am-test/api/management/apis/AlertsApi.ts

```diff
@@ -52,14 +52,14 @@ export interface GetAlertNotifierRequest {
     notifierId: string;
```

Comment:
This and every `ts` file in this `apis` directory is generated and can be ignored.